drivers/at86rf2xx/at86rf2xx.c

```diff
@@ -152,6 +229,16 @@ void at86rf2xx_reset(at86rf2xx_t *dev)
     /* clear interrupt flags */
     at86rf2xx_reg_read(dev, AT86RF2XX_REG__IRQ_STATUS);
 
+#if IS_USED(MODULE_IEEE802154_SECURITY) && \
+    IS_USED(MODULE_AT86RF2XX_AES_SPI)
+    dev->netdev.sec_ctx.dev.cipher_ops = &_at86rf2xx_cipher_ops;
+    dev->netdev.sec_ctx.dev.ctx = dev;
+    /* All configurations of the security module, the SRAM content,
+       and keys are reset during DEEP_SLEEP or RESET state. */
+        at86rf2xx_aes_key_write_encrypt(dev,
+            dev->netdev.sec_ctx.cipher.context.context);
```

Comment:
Unintentional indentation?

Reply:
I try to avoid lines longer than 80 characters because travis complains about that.

Reply:
Yes but `at86rf2xx_aes_key_write_encrypt(…` is still one level more indented than the code before it :wink:

Reply:
ah ... now I see